src/System.CommandLine/Binding/ArgumentConverter.DefaultValues.cs

```diff
@@ -25,14 +22,21 @@ private static Array CreateEmptyArray(Type itemType, int capacity = 0)
     private static IList CreateEmptyList(Type listType)
     {
 #if NET6_0_OR_GREATER
-        var ctor = (ConstructorInfo)listType.GetMemberWithSameMetadataDefinitionAs(_listCtor.Value);
+        ConstructorInfo? listCtor = _listCtor;
```

Comment:
Are there reflection-based argument conversions used with the source generator support?

Reply:
Source generator support is still pending. The original source generator work we did was superseded by improvements we've made to support trimming.